content/05.dl-complexities.md

```diff
@@ -1 +1,25 @@
 ## Tip 3: Understand the complexities of training deep neural networks {#complexities}
+
+Correctly training deep neural networks is a non-trivial process.
+There are many different options and potential pitfalls at every stage.
+To get good results you have to expect to train many networks with
+different tweaks and modifications.
+Despite improved framework ease-of-use and on-demand cloud computing resources
+this means DL can be very demanding, requiring significant
+infrastructure and patitence to complete.
+The experimentation inherent to DL is often noisy (requiring repetition) and
+represent a significant organisational challenge.
+All code, parameters, and results must be carefully corralled using good
+coding practices (e.g. version control, continuous integration etc.) in order to 
```

Comment:
e.g. and etc. shouldn't be used in the same statement. I suggest

- (e.g., version control and continuous integration)

or 

- (for example, version control and continuous integration)

Reply:
Fixed.